src/Juvix/Compiler/Concrete/Language.hs

```diff
@@ -1687,10 +1726,36 @@ makeLenses ''ArgumentBlock
 makeLenses ''NamedArgument
 makeLenses ''NamedApplication
 makeLenses ''AliasDef
+makeLenses ''FixitySyntaxDef
+makeLenses ''ParsedFixityInfo
+makeLenses ''ParsedFixityFields
+
+fixityAssoc :: SimpleGetter (ParsedFixityInfo s) (Maybe (BinaryAssoc))
+fixityAssoc = to (^? fixityFields . _Just . fixityFieldsAssoc . _Just)
+
+fixityPrecSame :: SimpleGetter (ParsedFixityInfo s) (Maybe (SymbolType s))
+fixityPrecSame = to (^? fixityFields . _Just . fixityFieldsPrecSame . _Just)
+
+fixityPrecAbove :: SimpleGetter (ParsedFixityInfo s) (Maybe [SymbolType s])
+fixityPrecAbove = to (^? fixityFields . _Just . fixityFieldsPrecAbove . _Just)
+
+fixityPrecBelow :: SimpleGetter (ParsedFixityInfo s) (Maybe [SymbolType s])
+fixityPrecBelow = to (^? fixityFields . _Just . fixityFieldsPrecBelow . _Just)
```

Comment:
They all look pretty similar except for the inner function fixityFields...
Curious if you factorise this somehow.

Reply:
done